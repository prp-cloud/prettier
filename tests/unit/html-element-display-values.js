import { htmlTagNames } from "html-tag-names";

import {
  CSS_DISPLAY_DEFAULT,
  CSS_DISPLAY_TAGS,
} from "../../src/language-html/constants.evaluate.js";

test("display", () => {
  const values = Object.fromEntries(
    htmlTagNames.map((tagName) => [
      tagName,
      CSS_DISPLAY_TAGS[tagName] ?? CSS_DISPLAY_DEFAULT,
    ]),
  );

  expect(values).toMatchSnapshot();
});
