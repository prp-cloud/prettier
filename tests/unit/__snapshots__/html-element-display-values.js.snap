// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display 1`] = `
{
  "a": "inline",
  "abbr": "inline",
  "acronym": "inline",
  "address": "block",
  "applet": "inline",
  "area": "none",
  "article": "block",
  "aside": "block",
  "audio": "inline-block",
  "b": "inline",
  "base": "none",
  "basefont": "none",
  "bdi": "inline",
  "bdo": "inline",
  "bgsound": "inline",
  "big": "inline",
  "blink": "inline",
  "blockquote": "block",
  "body": "block",
  "br": "inline",
  "button": "inline-block",
  "canvas": "inline",
  "caption": "table-caption",
  "center": "block",
  "cite": "inline",
  "code": "inline",
  "col": "table-column",
  "colgroup": "table-column-group",
  "command": "inline",
  "content": "inline",
  "data": "inline",
  "datalist": "none",
  "dd": "block",
  "del": "inline",
  "details": "block",
  "dfn": "inline",
  "dialog": "block",
  "dir": "block",
  "div": "block",
  "dl": "block",
  "dt": "block",
  "element": "inline",
  "em": "inline",
  "embed": "inline",
  "fieldset": "block",
  "figcaption": "block",
  "figure": "block",
  "font": "inline",
  "footer": "block",
  "form": "block",
  "frame": "inline",
  "frameset": "inline",
  "h1": "block",
  "h2": "block",
  "h3": "block",
  "h4": "block",
  "h5": "block",
  "h6": "block",
  "head": "none",
  "header": "block",
  "hgroup": "block",
  "hr": "block",
  "html": "block",
  "i": "inline",
  "iframe": "inline",
  "image": "inline",
  "img": "inline",
  "input": "inline-block",
  "ins": "inline",
  "isindex": "inline",
  "kbd": "inline",
  "keygen": "inline",
  "label": "inline",
  "legend": "block",
  "li": "list-item",
  "link": "none",
  "listing": "block",
  "main": "block",
  "map": "inline",
  "mark": "inline",
  "marquee": "inline-block",
  "math": "inline",
  "menu": "block",
  "menuitem": "inline",
  "meta": "none",
  "meter": "inline-block",
  "multicol": "inline",
  "nav": "block",
  "nextid": "inline",
  "nobr": "inline",
  "noembed": "none",
  "noframes": "none",
  "noscript": "inline",
  "object": "inline-block",
  "ol": "block",
  "optgroup": "block",
  "option": "block",
  "output": "inline",
  "p": "block",
  "param": "block",
  "picture": "inline",
  "plaintext": "block",
  "pre": "block",
  "progress": "inline-block",
  "q": "inline",
  "rb": "inline",
  "rbc": "inline",
  "rp": "none",
  "rt": "ruby-text",
  "rtc": "inline",
  "ruby": "ruby",
  "s": "inline",
  "samp": "inline",
  "script": "block",
  "search": "block",
  "section": "block",
  "select": "inline-block",
  "shadow": "inline",
  "slot": "contents",
  "small": "inline",
  "source": "block",
  "spacer": "inline",
  "span": "inline",
  "strike": "inline",
  "strong": "inline",
  "style": "none",
  "sub": "inline",
  "summary": "block",
  "sup": "inline",
  "svg": "inline",
  "table": "table",
  "tbody": "table-row-group",
  "td": "table-cell",
  "template": "inline",
  "textarea": "inline",
  "tfoot": "table-footer-group",
  "th": "table-cell",
  "thead": "table-header-group",
  "time": "inline",
  "title": "none",
  "tr": "table-row",
  "track": "block",
  "tt": "inline",
  "u": "inline",
  "ul": "block",
  "var": "inline",
  "video": "inline-block",
  "wbr": "inline",
  "xmp": "block",
}
`;
